pyanaconda/dbus/connection.py

```diff
@@ -63,6 +63,17 @@ def get_new_connection(self):
         """
```

Comment:
I guess that commit message should be:
``Make safe to observe services on buses that don't have to run``
not actual:
``Make is safe to observe services on buses that don't have to run``

Reply:
Fixed.